SourceX/miniwin/misc.cpp

```diff
@@ -178,13 +178,16 @@ HWND CreateWindowExA(
 	DvlIntSetting("grab input", &grabInput);
 
 #ifdef USE_SDL1
-	int flags = SDL_SWSURFACE | SDL_DOUBLEBUF | SDL_HWPALETTE;
+	int flags = SDL_SWSURFACE | SDL_HWPALETTE;
+#ifndef AMIGA
+	flags |= SDL_DOUBLEBUF;
+#endif
 	if (fullscreen) {
 		SDL_Log("fullscreen not yet supported with SDL1");
-	}
-	// flags |= fullscreen ? SDL_FULLSCREEN : SDL_RESIZABLE;
+	}	
+	flags |= fullscreen ? SDL_FULLSCREEN : SDL_RESIZABLE;

```

Comment:
@glebm could you test if this causes issues for your device, and if so maybe it needs a special case to ignore SDL_FULLSCREEN in the SDL1 code.